examples/pytorch_lightning/barlowtwins.py

```diff
@@ -41,11 +42,13 @@ def configure_optimizers(self):
 model = BarlowTwins()
 
 cifar10 = torchvision.datasets.CIFAR10("datasets/cifar10", download=True)
-dataset = LightlyDataset.from_torch_dataset(cifar10)
+dataset = LightlyDataset.from_torch_dataset(
+    cifar10, transform=SimCLRTransform(input_size=32)
```

Comment:
You're switching from `ImageCollateFunction` to `SimCLRTransform`. Is this intended?

Reply:
Yes, as in the old collate `SimCLRTransform` was inheriting from `ImageCollateFunction` and just changing some parameters

Reply:
Did you make sure the parameters are the same now as they were before or do we simply not care?